package.json

```diff
@@ -38,5 +22,29 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@mui-treasury/components": "^1.9.1",
+    "clsx": "^1.1.1",
+    "cra-template": "1.0.3",
+    "node-emoji": "^1.10.0",
+    "query-string": "^6.13.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "3.4.3",
+    "react-scroll-to-bottom": "^4.0.0",
+    "socket.io-client": "^2.3.0"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.14",
+    "@types/node-emoji": "^1.8.1",
+    "@types/react": "^16.9.49",
+    "@types/react-dom": "^16.9.8",
+    "@types/react-router-dom": "^5.1.5",
+    "@types/socket.io-client": "^1.4.34",
+    "typescript": "^4.0.3"
```

Comment:
When deploying code, only the generated files will be used (`js`). This means the only time the types are explicitly needed is when working with the pre-build files (`ts, tsx`), which is during development. As such, these dependencies are not truly `dependencies`, but rather `devDependencies`.